integrationtests/tests/fixtures/mod.rs

```diff
@@ -116,60 +117,161 @@ pub async fn fixtures(
     let (server_config, client_config) =
         ServerConfig::trusted_dealer_gen(&peers, max_evil, &params, OsRng::new().unwrap());
 
-    match env::var("FM_TEST_DISABLE_MOCKS") {
-        Ok(s) if s == "1" => {
-            info!("Testing with REAL Bitcoin and Lightning services");
-            let dir = env::var("FM_TEST_DIR").expect("Must have test dir defined for real tests");
-            let wallet_config = server_config.iter().last().unwrap().1.wallet.clone();
-            let bitcoin_rpc = bitcoincore_rpc::bitcoind_gen(wallet_config.clone());
-            let bitcoin = RealBitcoinTest::new(wallet_config);
-            let socket_gateway = PathBuf::from(dir.clone()).join("ln1/regtest/lightning-rpc");
-            let socket_other = PathBuf::from(dir).join("ln2/regtest/lightning-rpc");
-            let lightning =
-                RealLightningTest::new(socket_gateway.clone(), socket_other.clone()).await;
-            let lightning_rpc = Mutex::new(
-                ClnRpc::new(socket_gateway.clone())
-                    .await
-                    .expect("connect to ln_socket"),
-            );
-            let connect_gen = |cfg: &ServerConfig| TlsTcpConnector::new(cfg.tls_config()).to_any();
-            let fed = FederationTest::new(server_config.clone(), &bitcoin_rpc, &connect_gen).await;
-            let user_cfg = UserClientConfig(client_config.clone());
-            let user = UserTest::new(user_cfg.clone(), peers);
-            user.client.await_consensus_block_height(0).await;
-            let gateway = GatewayTest::new(
-                Box::new(lightning_rpc),
-                client_config.clone(),
-                lightning.gateway_node_pub_key,
-            )
-            .await;
+    let env_disable_mocks = env::var("FM_TEST_DISABLE_MOCKS").unwrap_or_default();
+    match env_disable_mocks.as_str() {
+        "1" => {
+            #[cfg(not(feature = "bitcoind"))]
+            info!("Testing with EXTERNAL bitcoind and EXTERNAL cln");
+            #[cfg(feature = "bitcoind")]
+            info!("Testing with INTERNAL bitcoind and EXTERNAL cln");
```

Comment:
I don't think we want the combinatorial explosion here since we'd need to test all combinations.